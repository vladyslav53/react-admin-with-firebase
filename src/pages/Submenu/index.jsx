import React from 'react';

const Submenu = () => <h1>Submenu</h1>;

export default Submenu;
