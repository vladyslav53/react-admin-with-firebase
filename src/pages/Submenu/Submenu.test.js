import React from 'react';
import { shallow } from 'enzyme';

import Submenu from '.';

describe('<Submenu /> rendering', () => {
  it('should render without crashing', () => {
    const component = shallow(<Submenu />);

    expect(component).toMatchSnapshot();
  });
});
