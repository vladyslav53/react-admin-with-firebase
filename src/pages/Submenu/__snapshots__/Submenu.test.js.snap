// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Submenu /> rendering should render without crashing 1`] = `
<h1>
  Submenu
</h1>
`;
